privaterelay/settings.py

```diff
@@ -80,6 +80,8 @@
 )
 REFERRER_POLICY = 'strict-origin-when-cross-origin'
 
+SITE_URL = config('SITE_URL', None)
```

Comment:
Seems like there is an env var for this now, from PR #209. See [here](https://github.com/mozilla/fx-private-relay/blob/master/privaterelay/settings.py#L103), there's a `SITE_ORIGIN` env var which got added to settings config.

Reply:
Oh yeah, good catch.